packages/cli/src/workflows/workflows.controller.ts

```diff
@@ -367,7 +366,11 @@ export class WorkflowsController {
 
 	@Post('/:workflowId/run')
 	@ProjectScope('workflow:execute')
-	async runManually(req: WorkflowRequest.ManualRun) {
+	async runManually(
+		req: WorkflowRequest.ManualRun,
+		_res: unknown,
```

Comment:
```suggestion

```

Reply:
The `_res` needs to be there to eat up the response, otherwise response is assigned to the query argument because its the second argument:

![image](https://github.com/user-attachments/assets/5a36df3f-8e1d-4b73-ac2c-4c0d3e2bb46e)
![image](https://github.com/user-attachments/assets/1f726a73-79f6-41cb-8807-dc1af71e69c9)

With `_res`:

![image](https://github.com/user-attachments/assets/b8b9dca4-7e95-41c1-88d8-b8fb3cfcef3a)
![image](https://github.com/user-attachments/assets/4ab74adf-ed14-4512-b18f-9c9324e15714)